### PR TITLE
CQ-4348870 Forms Core Component backport to 6.5

### PR DESCRIPTION
**Pull request by devgurjar**
@Review : @nitigupt @rismehta
Backported core component codes to 6.5

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: bundles/af-core/pom.xml [hunk 1]
```diff
@@ -159,7 +159,7 @@
                                                 <limit>
                                                     <counter>INSTRUCTION</counter>
                                                     <value>COVEREDRATIO</value>
-                                                    <minimum>0.80</minimum>
+                                                    <minimum>0.60</minimum>
```
Comment (nit23uec) [inline bundles/af-core/pom.xml:162]: pls log a jira for this.

File: bundles/af-core/pom.xml [hunk 2]
```diff
@@ -464,20 +464,19 @@
...
-
-
-        <dependency>
-            <groupId>com.adobe.aem</groupId>
-            <artifactId>aem-sdk-api</artifactId>
+            <groupId>com.adobe.aemfd</groupId>
+            <artifactId>aemfd-client-sdk</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
```
Comment (nit23uec) [inline bundles/af-core/pom.xml:471]: reason behind this?

**Comment by devgurjar** [comment]
@rismehta please review this PR

File: bundles/af-core/pom.xml (hunk 1 above)
Comment (devgurjar) [reply bundles/af-core/pom.xml:162]: https://jira.corp.adobe.com/browse/CQ-4351769

File: bundles/af-core/pom.xml (hunk 2 above)
Comment (rismehta) [reply bundles/af-core/pom.xml:471]: aemfd client SDK brings some old transitive dependencies which we don't need

**Pull request merged**